docs/en/ingest-management/tab-widgets/prereq.asciidoc

```diff
@@ -11,12 +9,18 @@ all spaces.
 
 // tag::self-managed[]
 
-* Access to an {es} cluster and {kib} (version {minor-version}) with a basic
-license. {stack-gs}/get-started-elastic-stack.html[Learn how to install the
+* An {es} cluster and {kib} (version {minor-version}) with a basic license or
+higher. {stack-gs}/get-started-elastic-stack.html[Learn how to install the
 {stack} on your own hardware].
 
 * Secure, encrypted connection between {kib} and {es}. For more information,
-see {ref}/configuring-stack-security.html[Configure security for the {stack}].
+see {ref}/configuring-stack-security.html[Start the {stack} with security enabled].
+
+* {kib} requires an internet connection to download integration packages from the
```

Comment:
I'm not going to make this parallel because I think it's more clear than using passive.

Reply:
Oh wait...I thought of a good edit.